app/models/concerns/orchestration/dhcp.rb

```diff
@@ -89,13 +89,11 @@ def boot_server
   end
 
   def boot_server_or_proxy_hostname
-    if tftp.bootServer.nil?
```

Comment:
Isn't changing this line to `unless tftp.bootServer.present?` the exact same thing? A minimal patch without any formatting changes is easier to cherry pick.

Reply:
That's extreme, I am almost sure it will apply cleanly on all versions and it looks nicer. Only if you insist.

Reply:
It's not just about applying cleanly. By doing it this way you do increase the review burden. What was otherwise trivial to review now requires mental parsing if you didn't make any other changes.